bin/authentication_milter

```diff
@@ -322,6 +322,8 @@ __END__
     "dns_servfail_timeout"  : 1000000,                  | How long (microseconds) a SERVFAIL can take before being considered a timeout
                                                         |     by the internal resolver
 
+    "hide_none"             : 0,                        | Do not add the Authentication-Results header if the result is 'none'
```

Comment:
hide_none feels like the wrong name for this, we're not hiding any 'none' results (like with the similarly named DMARC option), we are hiding the entire AR header.

Reply:
ah, but only if it would be 'none' gotcha